zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,100 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use zebra_chain::parameters::Network;
+
+/// Whatever
+pub struct Params(pub Network);
+
+impl Params {
+    /// COIN
+    pub const COIN: u32 = 100000000;
+
+    /// SlowStartInterval
+    pub const SLOW_START_INTERVAL: u32 = 20000;
+
+    /// MaxBlockSubsidy
+    pub const MAX_BLOCK_SUBSIDY: u32 = (12.5 * (Self::COIN as f32)) as u32;
```

Comment:
Avoid using floating-point values for exact money calculations.

If you can't use integer values, write a comment that explains why the floating-point result is exact. (Or why using the approximate value is ok.)

Sometimes you will need explanations for integer calculations as well.

```suggestion
    /// The largest possible block subsidy.
    ///
    /// This calculation is exact, because the division is executed last, and COIN is a multiple of 2.
    pub const MAX_BLOCK_SUBSIDY: Amount<NonNegative> = (25 * Self::COIN) / 2;
```